packages/purpose-process/src/services/readModelService.ts

```diff
@@ -375,24 +398,33 @@ export function readModelServiceBuilder(
 
       return result.data;
     },
-    async getActiveDelegation(
-      eserviceId: EServiceId,
-      kind: DelegationKind
+    async getActiveProducerDelegationByEserviceId(
```

Comment:
I did a small refactor on the getDelegation to align these with the other processes